packages/dashboard-frontend/src/components/BannerAlert/NoNodeAvailable/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2018-2024 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { api } from '@eclipse-che/common';
+import { render, RenderResult, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import BannerAlertNoNodeAvailable from '@/components/BannerAlert/NoNodeAvailable';
+import { container } from '@/inversify.config';
+import { WebsocketClient } from '@/services/backend-client/websocketClient';
+import { FakeStoreBuilder } from '@/store/__mocks__/storeBuilder';
+
+const websocketClient = container.get(WebsocketClient);
+const text =
+  'Cluster autoscaler is provisioning a new node at the moment. Please be patient, workspace startup will be taking longer than usual.';
```

Comment:
'FailedScheduling' event occurred. If cluster autoscaler is enabled it might be provisioning a new node now and workspace startup will take longer than usual. Check the 'Events' tab to get more details.

Reply:
done, changed the screenshot image.

Reply:
DId not add the `Check the 'Events' tab to get more details.` as the banner is also appears in the main dashboard page:
![screenshot-eclipse-che_apps_ci-ln-286d3lk-76ef8_aws-2_ci_openshift_org-2024_10_07-22_04_38](https://github.com/user-attachments/assets/49c7b74d-8197-4af8-afca-bcd2e11797fa)